android/src/main/java/com/revenuecat/purchases/common/common.kt

```diff
@@ -279,7 +279,18 @@ private fun getPurchaseErrorFunction(onResult: OnResult): (PurchasesError, Boole
     return { error, userCancelled -> onResult.onError(error.map(mapOf("userCancelled" to userCancelled))) }
 }
 
-private fun getPurchaseCompletedFunction(onResult: OnResult): (Purchase?, PurchaserInfo) -> Unit {
```

Comment:
I can't add a comment on the line because it wasn't changed, but line 266 on this file is the other place where compilation now fails because the symbol can't be found